sdk/storage/Azure.Storage.Blobs/src/Models/BlobLegalHoldInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Infomation about a blob's legal hold.
+    /// </summary>
+    public class BlobLegalHoldInfo
+    {
+        /// <summary>
+        /// If a legal hold is enabled on the blob.
```

Comment:
where do we explain what legal hold is ? some might be confused what's the difference between this and lease.

Reply:
We could add a link to the public rest docs when they are published.